### ISSUE TITLE
Long title for alert buttons

### ISSUE THREAD

**Issue by sergeygarazha** [huri000/SwiftEntryKit#121]
When there is 2 buttons in alert, and button title too long, string doesn't wraps to new line. Even if numberOfLines set to 2 for EKProperty.LabelStyle.

```swift
var attributes = EKAttributes.centerFloat
attributes.displayDuration = .infinity
attributes.windowLevel = .alerts
attributes.entryBackground = .visualEffect(style: .dark)
attributes.screenInteraction = .dismiss
attributes.hapticFeedbackType = .warning
attributes.position = .center
attributes.roundCorners = .all(radius: 12)
attributes.border = .value(color: UIColor.spDarkGray, width: 2)
attributes.screenBackground = .color(color: UIColor.black.withAlphaComponent(0.45))
attributes.positionConstraints.maxSize = EKAttributes.PositionConstraints.Size(width: EKAttributes.PositionConstraints.Edge.offset(value: 65), height: EKAttributes.PositionConstraints.Edge.intrinsic)
attributes.shadow = .active(with: EKAttributes.Shadow.Value(color: .black, opacity: 0.7, radius: 15, offset: .zero))

let messageImage = EKProperty.ImageContent(imageName: "image")
let titleLabelStyle = EKProperty.LabelStyle(font: UIFont.bold(20), color: UIColor.white, alignment: NSTextAlignment.center, numberOfLines: 0)
let titleLabelContent = EKProperty.LabelContent(text: title, style: titleLabelStyle)
let descriptionLabelStyle = EKProperty.LabelStyle(font: UIFont.systemFont(ofSize: 15), color: UIColor.white, alignment: NSTextAlignment.center, numberOfLines: 0)
let descriptionLabelContent = EKProperty.LabelContent(text: message, style: descriptionLabelStyle)
let message = EKSimpleMessage(image: messageImage, title: titleLabelContent, description: descriptionLabelContent)


let buttonStyle = EKProperty.LabelStyle(font: UIFont.regular(fontSize), color: titleColor, alignment: NSTextAlignment.center, numberOfLines: 0)
let labelContent = EKProperty.LabelContent(text: "Long button title", style: buttonStyle)
let firstButton = EKProperty.ButtonContent(label: labelContent, backgroundColor: backgroundColor, highlightedBackgroundColor: highlightedColor) {
    SwiftEntryKit.dismiss()
    action()
}

let buttonStyle = EKProperty.LabelStyle(font: UIFont.regular(fontSize), color: titleColor, alignment: NSTextAlignment.center, numberOfLines: 0)
let labelContent = EKProperty.LabelContent(text: "Cancel", style: buttonStyle)
let secondButton = EKProperty.ButtonContent(label: labelContent, backgroundColor: backgroundColor, highlightedBackgroundColor: highlightedColor) {
    SwiftEntryKit.dismiss()
    action()
}

let buttons = EKProperty.ButtonBarContent(with: firstButton, secondButton, separatorColor: UIColor.lightGray, expandAnimatedly: true)

let alertMessage = EKAlertMessage(simpleMessage: message, imagePosition: .top, buttonBarContent: buttons)
let view = EKAlertMessageView(with: alertMessage)
SwiftEntryKit.display(entry: view, using: attributes)
```

 - Device: [iPhone X, iPhone 8]
 - iOS Version: [iOS 12]
 - Xcode Version [10.0]
 - Dependency Manager Version [CocoaPods 1.5.3]
 - SwiftEntryKit Release # [0.7.2]

![fullsizeoutput_2050](https://user-images.githubusercontent.com/1624643/46535594-9b4d4100-c8b4-11e8-8834-112e76bd08cc.jpeg)

**Comment by huri000** [comment]
@sergeygarazha, Sorry for the late response. 
I'll do my best to get this issue and the bottom corners issue handled this week.

**Comment by huri000** [comment]
Sorry for the delay. Resolved in `0.8.6`.
In any case, the button title must have a reasonable length of text for a button.
Cheers!
🎉
